tests/test_gatk_preprocess.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+import pysam
+import pytest
+
+
+@pytest.mark.workflow("recalibrated_bam")
+def test_recalibrated_bam(workflow_dir):
```

Comment:
It might be good to add some comments on what exactly is getting tested. From the function name here it isn't clear.

Reply:
:+1: